_data/projects/Dive_into_Machine_Learning.yml

```diff
@@ -0,0 +1,12 @@
+name: Dive into Machine Learning
+desc: Dive into Machine Learning with Python Jupyter notebook and scikit-learn
+site: https://github.com/hangtwenty/dive-into-machine-learning
+tags:
+- machine-learning
+- ML
+- python
+- scikit-learn
+- data-science
+upforgrabs:
+  name: help%20wanted
```

Comment:
To be honest, I'm not 100% sure whether we'll encode this value when displaying it, but you can totally set `help-wanted` here instead